node/websocket/websocket_test.go

```diff
@@ -0,0 +1,87 @@
+package websocket_test
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/INFURA/go-ethlibs/node/websocket"
+)
+
+func getRopstenConnection(t *testing.T, ctx context.Context) websocket.Connection {
+	// These test require a ropsten websocket URL to test with, for example ws://localhost:8546 or wss://ropsten.infura.io/ws/v3/:YOUR_PROJECT_ID
+	url := os.Getenv("INFURA_TEST_ROPSTEN_WS_URL")
```

Comment:
For the sake of keeping the lib decoupled from infura, could just call the env var something like "ROPSTEN_WS_URL"

Reply:
It's `INFURA` as in the owner/namespace of the repo, not as in infura.io, just to namespace the environment var in case we ever want to implement something like https://www.reddit.com/r/golang/comments/ar5z3i/how_to_set_env_variables_while_unit_testing/egmzxnt/

Reply:
:+1:

Reply:
Could be `ETHLIBS_TEST_ROPSTEN_WS_URL` too, in case that sounds better. (Doesn't make a lot of sense to couple to the current author of the repo, especially with forks, etc.)